CHANGELOG.md

```diff
@@ -8,50 +8,64 @@
+- `<a-template>` and HTML imports logic have been removed. Use https://github.com/ngokevin/aframe-template-component in the meantime. (#883)
+- `<a-camera>` no longer creates a cursor on its own. Do `<a-camera><a-cursor></a-cursor></a-camera>` instead. (#883)
+- Default geometry `depth`, `height`, `width` property values changed from 2 to 1. (#1245)
+- Default color of primitive elements such as `<a-box>` changed to `#FFF` to not interfere with textures. (#1245)
 
 ### Deprecations
 
-* `loader` component deprecated in favor of `collada-model` and `obj-model`. (#913)
-* `<a-model>` deprecated in favor of `<a-collada-model>` and `<a-obj-model>`. (#883)
-* `<a-EVENTNAME>` elements such as `<a-mouseenter>` deprecated in favor of `<a-event name="EVENTNAME">` (unstable). (#883)
-* `<a-cube>` deprecated in favor of `<a-box>`. (#883)
+- `loader` component deprecated in favor of `collada-model` and `obj-model`. (#913)
+- `<a-model>` deprecated in favor of `<a-collada-model>` and `<a-obj-model>`. (#883)
+- `<a-EVENTNAME>` elements such as `<a-mouseenter>` deprecated in favor of `<a-event name="EVENTNAME">` (unstable). (#883)
+- `<a-cube>` deprecated in favor of `<a-box>`. (#883)
 
 ### Enhancements
 
-* Introduce *shaders* to extend the material component and to register custom GLSL shaders. (#861)
-* Component *property types*. Property types define how a component property is parsed and stringified. Custom property types can be registered or defined inline with the property in the schema. Built-in property types include `array`, `boolean`, `int`, `number`, `selector`, `selectorAll`, `string`, `vec2`, `vec3`, `vec4`. (d35e56e)
-* *Single-property components*. A component can define itself as consisting of only one property by specifying a type and/or a default value in the schema. (d35e56e)
-* *Asset management system* that blocks scene render
-* *Play/pause* methods on entities and play/pause handlers on components. (9238861)
-* *Tick* method on components to register a function called on each scene tick. (#823)
-* Support for loading *.OBJ* and .MTL assets. (#788)
-* *Texture caching* for better performance when reusing textures. (#1116)
-* Components can be attached to primitives (e.g., `<a-sphere>`). (#883)
-* Introduce *systems* API (unstable) to provide global scope and services for components. (#924)
-* Entities, including the scene, wait for their children to load before emitting the `loaded` event. (a8a4f06)
-* Entities emit `child-attach` when children are attached.
-* Lot of `<a-scene>` logic moved to configurable components. (#776)
-* Support for *multiple cameras* in a scene and switching between them. (#745)
-* Added more events for scene VR mode, material component, model components, sound component.
+- Introduce *shaders* to extend the material component and to register custom GLSL shaders. (#861)
```

Comment:
should probably use past tense to be consistent with usage elsewhere:

`Introduce` -> `Introduced`